pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -202,11 +203,11 @@ let
 
       env = callPackage ../qt-env.nix {};
       full = env "qt-full-${qtbase.version}" ([
-        qt3d qtcharts qtconnectivity qtdeclarative qtdoc qtgraphicaleffects
-        qtimageformats qtlocation qtmultimedia qtquickcontrols qtquickcontrols2
-        qtscript qtsensors qtserialport qtsvg qttools qttranslations
-        qtvirtualkeyboard qtwebchannel qtwebengine qtwebkit qtwebsockets
-        qtwebview qtx11extras qtxmlpatterns qtlottie
+        qt3d qtcharts qtconnectivity qtdatavis3d qtdeclarative qtdoc
```

Comment:
Not sure if that should be part of full or not. Also I wouldn't reflow here to reduce the diff.

Reply:
I can't judge about "full or not", but I undid the reflow to reduce the diff.